SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Windows.Shared/Crashes.cs

```diff
@@ -386,8 +386,13 @@ private void InstanceTrackError(System.Exception exception, IDictionary<string,
                 }
                 properties = PropertyValidator.ValidateProperties(properties, "HandledError");
                 var exceptionAndBinaries = ErrorLogHelper.CreateModelExceptionAndBinaries(exception);
-                var log = new HandledErrorLog(exception: exceptionAndBinaries.Exception, binaries:exceptionAndBinaries.Binaries, properties: properties, id: Guid.NewGuid(), device: null, userId: UserIdContext.Instance.UserId); ;
+                var errorId = Guid.NewGuid();
+                var log = new HandledErrorLog(exception: exceptionAndBinaries.Exception, binaries:exceptionAndBinaries.Binaries, properties: properties, id: errorId, device: null, userId: UserIdContext.Instance.UserId);
```

Comment:
```suggestion
                var log = new HandledErrorLog(exception: exceptionAndBinaries.Exception, binaries:exceptionAndBinaries.Binaries, properties: properties, id: errorId, userId: UserIdContext.Instance.UserId);
```

Reply:
@guperrot we don't seem to have a constructor for this suggestion.

Reply:
I think we should add it. Cleaner API usage :)

Reply:
@guperrot `device` parameter doesn't have a default `null` value so passing `device: null` to the constructor is required.

Reply:
Ok, @jwallra this is generated code so no.